contrib/epee/src/readline_buffer.cpp

```diff
@@ -167,30 +165,42 @@ static int handle_enter(int x, int y)
   char* line = NULL;
```

Comment:
There is an underused custom deleter argument that would require a couple of lines in this case, and has no additional overhead (type erasure/allocations, etc.). I think this is the first place that needed a `free(...)` deleter; I added some terrible looking Windows specific free calls somewhere that cannot be reused here.